src/main/java/seedu/address/model/LogBook.java

```diff
@@ -0,0 +1,159 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.UniquePersonList;
+
+/**
+ * Wraps all data at the address-book level
+ * Duplicates are not allowed (by .isSamePerson comparison)
+ */
+public class LogBook implements ReadOnlyAddressBook {
+
+    private final UniquePersonList loggedPersons;
+
+    /*
+     * The 'unusual' code block below is a non-static initialization block,
+     * sometimes used to avoid duplication
+     * between constructors. See
+     * https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html
+     *
+     * Note that non-static init blocks are not recommended to use. There are other
+     * ways to avoid duplication
+     * among constructors.
+     */
+    {
+        loggedPersons = new UniquePersonList();
+        // placeholder data (commented for the sake of test case), constructor should start with an empty list
+        // loggedPersons.add(
+        //         new Person(new Name("Alex Yeoh"), new Nric("T0123456F"), new Phone("87438807"),
+        //                 new Email("alexyeoh@example.com"),
+        //                 new Address("Blk 30 Geylang Street 29, #06-40"),
+        //                 new Appointment("2023-08-08 10:00 12:00"),
+        //                 SampleDataUtil.getMedicalHistorySet(),
+        //                 SampleDataUtil.getTagSet("friends")));
+        // loggedPersons.add(
+        //         new Person(new Name("Bernice Yu"), new Nric("S0123452F"), new Phone("99272758"),
+        //                 new Email("berniceyu@example.com"),
+        //                 new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
+        //                 new Appointment("2023-08-08 12:00 14:00"),
+        //                 SampleDataUtil.getMedicalHistorySet("Diabetes"),
+        //                 SampleDataUtil.getTagSet("colleagues", "friends")));
+        // loggedPersons.add(
+        //         new Person(new Name("Charlotte Oliveiro"), new Nric("T0123456Y"), new Phone("93210283"),
+        //                 new Email("charlotte@example.com"),
+        //                 new Address("Blk 11 Ang Mo Kio Street 74, #11-04"),
+        //                 null,
+        //                 SampleDataUtil.getMedicalHistorySet("AB+ Blood"),
+        //                 SampleDataUtil.getTagSet("neighbours")));
```

Comment:
Keeping unused code in a comment like this violates our coding standards!

Reply:
This was intended for easier tracing for other members to test the logger tab. Perhaps I could just leave the line to indicate where placeholder data could be inserted and remove this chunk of unused code. Thank you!